src/librustdoc/html/static/main.js

```diff
@@ -1490,27 +1490,42 @@ function hideThemeButtonState() {
     searchState.setup();
 }());
 
-function copy_path(but) {
-    var parent = but.parentElement;
-    var path = [];
+(function () {
+    var reset_button_timeout = null;
 
-    onEach(parent.childNodes, function(child) {
-        if (child.tagName === 'A') {
-            path.push(child.textContent);
-        }
-    });
+    function copy_path(but) {
```

Comment:
You need to put this function into `window`, otherwise the `onclick` event won't find it.

```js
window.copy_path = copy_path;
```

Or you can declare it directly like this:

```suggestion
    window.copy_path = function(but) {
```

(and don't forget the `;` at the end ;) )